resources/charts/commander/templates/pod.yaml

```diff
@@ -12,21 +12,19 @@ spec:
     - name: {{ .Chart.Name }}
       image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
       imagePullPolicy: {{ .Values.image.pullPolicy }}
-      command: ["/bin/sh", "-c"]
+      command: ["/entrypoint.sh"]
```

Comment:
Command is already specified in the dockerfile as `ENTRYPOINT`. Is this line required?